### PR TITLE
Add hyperterm-summon

### PR DESCRIPTION
**Pull request by zigomir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 * [hyperterm-sync-settings](https://www.npmjs.com/package/hyperterm-sync-settings) - Easy way to backup and restore HyperTerm settings to Github.
 * [hyperterm-mactabs](https://www.npmjs.com/package/hyperterm-mactabs) - Better tab styles, with macOS-inspired design and close buttons on the left, compatible with most themes.
 * [hyperterm-final-say](https://www.npmjs.com/package/hyperterm-final-say) - Allows user-set overrides of any plugin or theme settings applied on top of the defaults `./.hyperterm.js`.
+* [hyperterm-summon](https://github.com/soutar/hyperterm-summon) - Summon your Hyperterm windows with a system-wide hotkey.
```
Comment (matheuss) [inline README.md]: Could you change the url to `https://www.npmjs.com/package/hyperterm-summon`? Thanks ☺️

**Comment by matheuss** [comment]
I use that a lot in iTerm ☺️

**Comment by zigomir** [comment]
@matheuss done ;)

**Comment by matheuss** [comment]
Thanks for the contribution and sorry for the delay 😅

**Pull request merged**